test/registration-login.spec.ts

```diff
@@ -6,25 +6,27 @@ dotenv.config({path: `.${env.get('NODE_ENV').required().asString()}.env`});
 
 describe('registration . login', () => {
   let tokenAdmin: string;
-
+  const host = env.get('HOST').required().asString();
+  const port = env.get('PORT').required().asString();
+  
```

Comment:
why you keep duplicating that code, create separate file -> compose the string -> export it -> import in tests